package.json

```diff
@@ -69,7 +69,8 @@
   },
   "peerDependencies": {
     "@edx/frontend-platform": "^7.0.0 || ^8.0.0",
-    "@openedx/paragon": ">= 21.5.7 < 23.0.0",
+    "@openedx/paragon": "^22.8.0",
```

Comment:
This was intentional as it was built on top of Paragon `22.8.0`, but it will be backward compatible, so I reverted it here: ccee347905da3fc899dbdfdcb29066f73a1aec8d

But I still think we should update the version here, and I can't see how this could impact someone, as the current `package-lock.json` is already pointing to `22.8.1`.

https://github.com/openedx/frontend-component-header/blob/e44001e94553d377a60f341028394679b2ab06df/package-lock.json#L3407-L3408

Could you help me to understand?

Thank you!

Reply:
Also, I'm not sure if we should have `@openedx/paragon` (and some other packages such as `react`, `react-dom`, etc..) in the `devDependencias`, but I think this is an issue for another PR. :smiley:

Reply:
The `package-lock.json` is pointing to 22.8.1 due to `@openedx/paragon` getting installed as a `devDependency`, which is intentional. Because this library relies on consumers to provide their own copy of `@openedx/paragon` (per  the `peerDependencies`), the `devDependencies` copy of `@openedx/paragon` is what allows for local development of the library. Without Paragon in the `devDependencies`, the example app within this repo couldn't run due to the missing installed `node_modules/@openedx/paragon`.

The `devDependencies` are not included in the production bundle published to NPM, as confirmed by noting that `@openedx/paragon` is included only in the `npm list --include=dev` command, but not `npm list --omit=dev` as seen below:

```shell
❯ npm list --omit=dev --depth=0
@edx/frontend-component-header@1.0.0-semantically-released /Users/astankiewicz/Desktop/edx/frontend-component-header
├── @fortawesome/fontawesome-svg-core@6.6.0
├── @fortawesome/free-brands-svg-icons@6.6.0
├── @fortawesome/free-regular-svg-icons@6.6.0
├── @fortawesome/free-solid-svg-icons@6.6.0
├── @fortawesome/react-fontawesome@0.2.2
├── @openedx/frontend-plugin-framework@1.3.0
├── axios-mock-adapter@1.22.0
├── babel-polyfill@6.26.0
├── classnames@2.5.1
├── jest-environment-jsdom@29.7.0
├── react-responsive@8.2.0
└── react-transition-group@4.4.5

❯ npm list --include=dev --depth=0
@edx/frontend-component-header@1.0.0-semantically-released /Users/astankiewicz/Desktop/edx/frontend-component-header
├── @edx/brand@npm:@openedx/brand-openedx@1.2.3
├── @edx/browserslist-config@1.2.0
├── @edx/frontend-platform@8.1.1
├── @edx/reactifex@2.2.0
├── @fortawesome/fontawesome-svg-core@6.6.0
├── @fortawesome/free-brands-svg-icons@6.6.0
├── @fortawesome/free-regular-svg-icons@6.6.0
├── @fortawesome/free-solid-svg-icons@6.6.0
├── @fortawesome/react-fontawesome@0.2.2
├── @openedx/frontend-build@14.1.5
├── @openedx/frontend-plugin-framework@1.3.0
├── @openedx/paragon@22.8.1
├── @testing-library/dom@10.4.0
├── @testing-library/jest-dom@5.17.0
├── @testing-library/react@10.4.9
├── axios-mock-adapter@1.22.0
├── babel-polyfill@6.26.0
├── classnames@2.5.1
├── husky@8.0.3
├── jest-chain@1.1.6
├── jest-environment-jsdom@29.7.0
├── jest@29.7.0
├── prop-types@15.8.1
├── react-dom@17.0.2
├── react-redux@7.2.9
├── react-responsive@8.2.0
├── react-router-dom@6.26.2
├── react-test-renderer@17.0.2
├── react-transition-group@4.4.5
├── react@17.0.2
├── redux-saga@1.3.0
└── redux@4.2.1
```

Reply:
Now I got it! Thank you for the explanation!